pdf2image/pdf2image.py

```diff
@@ -24,7 +24,7 @@
     PDFSyntaxError
 )
 
-def convert_from_path(pdf_path, dpi=200, output_folder=None, first_page=None, last_page=None, fmt='ppm', thread_count=1, userpw=None, use_cropbox=False, strict=False, transparent=False):
+def convert_from_path(pdf_path, dpi=200, output_folder=None, first_page=None, last_page=None, fmt='ppm', thread_count=1, userpw=None, use_cropbox=False, strict=False, transparent=False, out_file=None):
```

Comment:
Change `out_file=None` to `out_file=str(uuid.uuid4())` to avoid breaking changes with previous behavior.

Reply:
Also, pdftocairo calls it `output_file` instead of `out_file` so you could go with that instead for added clarity.